kafka-rest/src/main/java/io/confluent/kafkarest/v2/KafkaConsumerManager.java

```diff
@@ -129,7 +126,7 @@ public void rejectedExecution(Runnable r, ThreadPoolExecutor executor) {
                   + "The thread pool must be satured or shutiing down", r);
```

Comment:
maybe: `The runnable {} was rejected execution because the thread pool is saturated or shutting down`